torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -82,6 +82,10 @@ XLATensor GetOrCreateXlaTensor(const at::Tensor& tensor, const Device& device) {
   return xtensor ? *xtensor : XLATensor::Create(tensor, device);
 }
 
+XLATensor GetXlaTensorOrNull(const at::Tensor& tensor) {
```

Comment:
This functions is very similar to `GetOrCreateXlaTensor`, is it intended?

Reply:
The difference is that if `tensor` is defined, it must be an XLA tensor.
But now that I think more about those curve ball coming from PT, with tensors from other devices being passed, we might just use the other one.